channels/mx.m3u

```diff
@@ -66,10 +66,6 @@ https://mdstrm.com/live-stream-playlist_400/57b4dbf5dbbfc8f16bb63ce1.m3u8
 http://dcunilive21-lh.akamaihd.net/i/dclive_1@59479/index_1_av-p.m3u8
 #EXTINF:-1 tvg-id="OnceInternacional.mx" tvg-name="Once Internacional" tvg-country="MX" tvg-language="Spanish" tvg-logo="https://i.imgur.com/NAgRCyY.png" group-title="",Once Internacional [Offline]
 http://live.canaloncelive.tv:1935/livepkgr2/smil:internacional.smil/playlist.m3u8
```

Comment:
This link also have the same issue than the ones below.

Reply:
The script always leaves the last link to the channel in the playlist, even if it does not work, to keep the channel description.

Reply:
I see. I'm only indicating it for informational purposes :)